### PR TITLE
I solved hw7 objects in memory task4 Tv and Tv Controller

### PR DESCRIPTION
**Pull request by avp02**

### CODE REVIEW

**Review by kashlyak — COMMENTED** [review]
lgfm

**Review by olgaZskiba — APPROVED** [review]
Lgfm

File: src/test/java/com/avp/homework7objectsinmemory/task4tvandtvconroller/service/TvControllerServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    @BeforeEach
+    void beforeEach() {
+        testTvController = new TvController();
+        test = new TvControllerService(testTvController);
+        testTvController.setCurrentChannel(3);
+        testTvController.setCurrentSound(15);
+    }
+
+    @Test
+    void IncreaseSound() {
```
Comment (lillieMaiBauer) [inline src/test/java/com/avp/homework7objectsinmemory/task4tvandtvconroller/service/TvControllerServiceTest.java]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

**Comment by avp02** [comment]
I agreed

File: src/test/java/com/avp/homework7objectsinmemory/task4tvandtvconroller/service/TvControllerServiceTest.java (hunk 1 above)
Comment (avp02) [reply src/test/java/com/avp/homework7objectsinmemory/task4tvandtvconroller/service/TvControllerServiceTest.java]: I agree I didn't notice

**Review by akupreychik — APPROVED** [review]
LGFM

**Pull request merged**